### PR TITLE
Build fix: replace distribution method with fastlane hockey

### PR DESCRIPTION
**Pull request by Xingyuj**
replace distribution method with fastlane hockey

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -53,28 +51,26 @@ pushd .
...
-                    -app_id=$HOCKEYAPP_APPID_DYNAMIC \
-                    $BUILD_OUTPUTS/PZDynamic.ipa
-/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" \
-                    -notes_type=markdown \
-                    -source_path=$(pwd) \
-                    -repository_url=https://github.com/StreetHawkSDK/ios \
-                    -api_token=$HOCKEYAPP_TOKEN \
-                    -app_id=$HOCKEYAPP_APPID_STATIC \
-                    $BUILD_OUTPUTS/PZStatic.ipa
+#/usr/local/bin/puck -submit=auto -download=true -notes="$(git log -1)" \
```
Comment (jagguli) [inline build.sh]: why is this commented ?

Comment (Xingyuj) [reply build.sh]: it is replaced by a method hockey() in "fastlane beta"

Comment (Xingyuj) [reply build.sh]: do you want me to delete it, in stead of comment out?

Comment (jagguli) [reply build.sh]: remove it . and are you using the env variables ?

**Pull request merged**